packages/tokens/src/bin/Generators/SassGenerator.spec.ts

```diff
@@ -28,25 +28,23 @@ describe("SassGenerator", () => {
       "$colors: (
         'primary': #055FD2,
         'secondary': #DA0000,
-        'ternary': (
-          '900': #022858
-        )
-      );
+        'ternary-900': #022858
```

Comment:
I would add a test case for `dark-greyis-thing` ( snake case format )

Reply:
I add something 👹